server/channels/api4/team_test.go

```diff
@@ -1154,6 +1155,12 @@ func TestGetAllTeams(t *testing.T) {
 				return
 			}
 			require.NoError(t, err2)
+
+			sort.Strings(tc.ExpectedTeams)
+			sort.Slice(teams, func(i, j int) bool {
+				return teams[i].Id < teams[j].Id
+			})
```

Comment:
Fix non-determinism in assertions.

Reply:
Nit: wonder if we could use the [ElementsMatch](https://pkg.go.dev/github.com/stretchr/testify/require#Assertions.ElementsMatch) helper for these cases

Reply:
Oh, YES! I looked for such a method, but couldn't grep with any keywords that made sense to me so I gave up. Thank you :D